cmd/olm/manager.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
```

Comment:
interesting that controller runtime has a zap log pkg - is it related to the uber zap logger?

Reply:
Looks like it is